android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -102,8 +103,26 @@ class RNPlayerView(
     private val activityLifecycle = (context.currentActivity as? ReactActivity)?.lifecycle
         ?: error("Trying to create an instance of ${this::class.simpleName} while not attached to a ReactActivity")
 
+    /**
+     * Relays the provided set of events, emitted by the player, together with the associated name
+     * to the `eventOutput` callback.
+     */
+    private var playerEventRelay: EventRelay<Player, Event>
+
+    private var playerInMediaSessionService: Player?
+        get() = context.playerModule?.mediaSessionPlaybackManager?.serviceBinder?.player
+        set(value) {
+            value?.let {
```

Comment:
Don't we want to set the player in service binder to null if we set playerInMediaSessionService to null like on the line 139? With this `playerInMediaSessionService = null` has no effect.

Reply:
You're right, nice catch! 💪 
Address in abaedf2fe5aedfd66baa95d12785e257ea35c0b4